yearn/yearn.py

```diff
@@ -52,23 +53,43 @@ def __init__(self, load_strategies=True, load_harvests=False, watch_events_forev
             self.registries["v2"].load_harvests()
         logger.info('loaded yearn in %.3fs', time() - start)
 
+
+    def active_vaults_at(self, block=None):
+        active = set()
+        registries = self.registries.items()
+        for label, registry in registries:
+            if label == 'earn':
+                active = active.union({vault.vault for vault in registry.active_vaults_at_block(block=block)})
+            else:
+                active = active.union({vault.vault for vault in registry.active_vaults_at(block=block)})
+        try:
+            active.remove(contract("0xBa37B002AbaFDd8E89a1995dA52740bbC013D992")) # [yGov] Doesn't count for this context
+        except KeyError:
+            pass
+        return active
+    
+    
     def describe(self, block=None):
         desc = Parallel(4, "threading")(
             delayed(self.registries[key].describe)(block=block)
             for key in self.registries
         )
         return dict(zip(self.registries, desc))
 
+
     def describe_wallets(self, block=None):
-        registries = ['v1','v2'] # TODO: add other registries [earn, ib, special]
         describer = RegistryWalletDescriber()
-        data = Parallel(4,'threading')(delayed(describer.describe_wallets)(self.registries[key], block=block) for key in registries)
-        data = {registry:desc for registry,desc in zip(registries,data)}
+        data = Parallel(4,'threading')(delayed(describer.describe_wallets)(registry, block=block) for registry in self.registries.items())
+        data = {registry:desc for registry,desc in zip(self.registries,data)}
 
         wallet_balances = Counter()
         for registry, reg_desc in data.items():
-            for wallet, usd_bal in reg_desc['wallet balances usd'].items():
-                wallet_balances[wallet] += usd_bal
+            try:
+                for wallet, usd_bal in reg_desc['wallet balances usd'].items():
+                    wallet_balances[wallet] += usd_bal
+            except:
```

Comment:
use a more specific `except` here

Reply:
this was just an artifact left over from testing, will remove